google/cloud/forseti/services/utils.py

```diff
@@ -258,3 +259,20 @@ def get_resource_id_from_type_name(type_name):
     if '/' in type_name:
         return type_name.split('/')[-1]
     return type_name
+
+
+def opencensus_enabled():
+    """Check if `opencensus` instrumentation library can be imported.
+
+    Returns:
+        bool: True if check pass, False otherwise.
+    """
+    # Check if OpenCensus exist on system
+    try:
+        imp.find_module('opencensus')
+        return True
+    except ImportError:
+        logging.warning('Cannot enable tracing because the `opencensus` library\
```

Comment:
Backslash should not be used for line cotinuations.
```
logging.warning(
    'Cannot enable tracing because the `opencensus` library'
    'was not found. Run `pip install .[tracing]` to install tracing libraries.')